src/modules/historical.ts

```diff
@@ -80,6 +80,44 @@ export default function historical(
 
     result: {
       schemaKey: "#/definitions/HistoricalResult",
+      transformWith(result: any) {
+        const filteredResults = [];
+        const fieldCount = Object.keys(result[0]).length;
+
+        // Count number of null values in object (1-level deep)
+        function nullFieldCount(object: Object) {
+          let nullCount = 0;
+          for (let val of Object.values(object)) if (val === null) nullCount++;
```

Comment:
nit: const val

Reply:
thanks, @okomarov, this is an old bad habit of mine that I seem to always slip back into without realising.  so thanks for that.